stylesheets/application.css

```diff
@@ -292,26 +294,31 @@ nav > .play-cta { margin: 0; padding: 10px 20px; color: white; }
 .contact > *:last-child:hover { background: rgb(117, 115, 190); }
 .contact > *:last-child:active { background: rgb(142, 140, 215); }
 
+.version { display: flex; flex-direction: row; }
+
+.version-btn { margin: 0px 20px; }
+.version-btn h2 { margin-bottom: 0px; }
```

Comment:
Is this actually used somewhere? An h2 should not be inside a button. It doesn't look like you're actually doing that on the page though?